code/modules/research/designs/medals.dm

```diff
@@ -0,0 +1,89 @@
+/datum/design/medal/conduct
+	name = "distinguished conduct medal"
+	desc = "A bronze medal awarded for distinguished conduct. Whilst a great honor, this is most basic award given by Nanotrasen. It is often awarded by a captain to a member of his crew."
+	id = "medal_conduct"
+	build_type = AUTOLATHE
```

Comment:
these are supposed to all get a new define, see designs.dm line 55 to add your own

Reply:
changed